.github/workflows/mac.yml

```diff
@@ -623,9 +642,9 @@ jobs:
       - name: MO Python API Tests
         run: |
           python3 -m pip install -r ${{ env.LAYER_TESTS_INSTALL_DIR }}/requirements.txt
-          export PYTHONPATH=${{ env.LAYER_TESTS_INSTALL_DIR }}:$PYTHONPATH
-          # TODO: remove setupvars.sh from here; currently, it's used for 'test_utils' installed in '<package>/python/openvino'
-          source ${INSTALL_DIR}/setupvars.sh
+          
+          # Used for 'test_utils' installed in '<test_package>/python/openvino/test_utils'
+          export PYTHONPATH=${{ env.INSTALL_TEST_DIR }}/python/openvino/test_utils:$PYTHONPATH
```

Comment:
Should we use `test_utils` from the test package or from the installed OV wheels?

Reply:
Installed OV wheels contain product part of OpenVINO and don't contain any test utils.
So, we need to use it from test package

Reply:
are `test_utils` even in the wheels?

Reply:
there is no `test_utils` module in wheel

Reply:
My bad, did not update the page after the first comment, thanks for the clarification.